pennylane/transforms/zx/reducer.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2018-2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module provides transforms to reduce a circuit applying simplification rules based on the ZX calculus.
+For more details about ZX calculus-based simplification of quantum circuits, see the following papers:
+
+    - Ross Duncan, Aleks Kissinger, Simon Perdrix, John van de Wetering (2019),
+    "Graph-theoretic Simplification of Quantum Circuits with the ZX-calculus", <https://arxiv.org/abs/1902.03178>
+
+    - Aleks Kissinger, John van de Wetering (2020),
+    "Reducing T-count with the ZX-calculus", <https://arxiv.org/abs/1903.10477>
+
+For the list of ZX calculus-based simplification rules implemented in ``pyzx``, see the online documentation:
+<https://pyzx.readthedocs.io/en/latest/api.html#list-of-simplifications>
```

Comment:
This is actually quite useful information.  Can we actually move it to the docstring instead so that users have access to it as well?